nmdc_runtime/api/endpoints/find.py

```diff
@@ -176,44 +176,46 @@ def find_data_object_by_id(
 
 
 @router.get(
-    "/activities",
+    "/workflow_executions",
     response_model=FindResponse,
     response_model_exclude_unset=True,
 )
-def find_activities(
+def find_workflow_executions(
     req: FindRequest = Depends(),
     mdb: MongoDatabase = Depends(get_mongo_db),
 ):
+    # TODO: Update description to be accurate wrt Berkeley schema
     """
-    The GET /activities endpoint is a general way to fetch metadata about various activities (e.g. metagenome assembly,
+    The GET /workflow_executions endpoint is a general way to fetch metadata about various workflow_executions (e.g. metagenome assembly,
     natural organic matter analysis, library preparation, etc.). Any "slot" (a.k.a. attribute) for
-    [WorkflowExecutionActivity](https://microbiomedata.github.io/nmdc-schema/WorkflowExecutionActivity/)
+    [WorkflowExecution](https://microbiomedata.github.io/nmdc-schema/WorkflowExecution/)
```

Comment:
Does not currently resolve. Ideally we'd take this opportunity to use our intended persistent ID here, i.e. <https://w3id.org/nmdc/WorkflowExecution>.

For now, use <https://microbiomedata.github.io/berkeley-schema-fy24/WorkflowExecution> and leave a note to refactor to use the `w3id.org` URL once that is active.

Reply:
If updating to the berkeley fork please make a ticket/note/todo about updating to nmdc-schema ahead of the official rollout.